pom.xml

```diff
@@ -869,6 +865,41 @@
 				</repository>
 			</repositories>
 		</profile>
+		<profile>
+			<!-- By default, we release artifacts to Sonatype, which requires
+			  nexus-staging-maven-plugin. Going forward, we'll use pure
+			  maven-deploy-plugin, and we need to turn this extension off. -->
+			<id>release-sonatype</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.sonatype.plugins</groupId>
+						<artifactId>nexus-staging-maven-plugin</artifactId>
+					</plugin>
+				</plugins>
+			</build>
+		</profile>
+		<profile>
+			<!-- Optionally, we can publish the artifacts to GCP Artifact Registry specifying
+			  this release-gcp-artifact-registry profile:
+			  mvn deploy -P=release-gcp-artifact-registry -P=-release-sonatype \
```

Comment:
Is `-P=-release-sonatype` still needed if we want to release to artifact registry?

Reply:
No, it's not needed but it's just for clarity.